stable/msoms/templates/omsagent-daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+ name: {{ .Values.agentname }}
+spec:
+ updateStrategy:
+    type: RollingUpdate
+ template:
+  metadata:
+   labels:
```

Comment:
Only `app` and `release` labels should go here.

Reply:
This has been addressed